simpleSequential/test_stringToInteger.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+from stringToInteger import stringToInteger
+
+class TestStringToInteger(unittest.TestCase):
```

Comment:
Great test. But I think instead of “positiveStringInput”. Let use “positiveNumericalStringInput”. Same for the 2 other tests.

Reply:
Did you read this one?

Reply:
Acknowledge.

Reply:
I think you fix the wrong file. Did you check the file name?